server/sonar-db-dao/src/main/java/org/sonar/db/rule/RuleDao.java

```diff
@@ -36,6 +36,14 @@
 
 public class RuleDao implements Dao {
 
+  public Optional<RuleDto> selectByKey(DbSession session, OrganizationDto organization, RuleKey key) {
+    return selectByKey(session, organization.getUuid(), key);
+  }
+
+  /**
+   * @deprecated use {@link #selectByKey(DbSession, OrganizationDto, RuleKey)}
+   */
+  @Deprecated
   public Optional<RuleDto> selectByKey(DbSession session, String organizationUuid, RuleKey key) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)